libs/providers/flagd-web/src/lib/options.ts

```diff
@@ -0,0 +1,53 @@
+export interface Options {
```

Comment:
What about the path? We may need to support at least a path prefix.

Reply:
I will look into this.

Reply:
I've added `pathPrefix` and tested it!